tools/pipelines/build-docs.yml

```diff
@@ -301,10 +301,30 @@ stages:
             workingDir: $(Build.SourcesDirectory)/docs
             customCommand: 'run ci:linkcheck'
 
+- stage: check_branch_version
+  displayName: 'Check Version Deployment Condition'
+  pool: Small
+  jobs:
+    - job: check_branch_version
+      displayName: 'Check Version Deployment Condition'
+      steps:
+        - task: Bash@3
+          displayName: 'Check Version Deployment Condition'
+          inputs:
+            targetType: 'inline'
+            workingDirectory: ${{ parameters.buildDirectory }}
+            script: |
+
+              # Generate the build version. Sets the environment variables version, codeVersion, and isLatest.
+              flub generate buildVersion
```

Comment:
Double-check that the include-vars.yml workflow is included in this pipeline. Those vars are all required for generate:buildVersion to work correctly.